bin/auto_hck

```diff
@@ -53,6 +53,7 @@ Signal.trap('TERM') do
 end
 
 Thread.abort_on_exception = true
+Thread.report_on_exception = false
```

Comment:
"report_exception?" probably better

Reply:
This only prevents the thread exception to be reported to stderr, the exception will still be raised from the thread to the main thread.